### PR TITLE
INTLY-7427 Add test for PVC check

### PR DESCRIPTION
**Pull request by CathalOConnorRH**
# Description
Add test to check for PVC being available and bound. This test is to replace manual testing. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: test/common/pvc_validate.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+func TestPVClaims(t *testing.T, ctx *TestingContext) {
+
+	//get full list of volume claim items
+	pvcs := &corev1.PersistentVolumeClaimList{}
+
+	for _, pvcNamespace := range pvcNamespaces {
+		err := ctx.Client.List(goctx.TODO(), pvcs, &k8sclient.ListOptions{Namespace: pvcNamespace.Namespace})
+		if err != nil {
+			t.Errorf("Error getting PVCs for namespace: %v %w", pvcNamespace.Namespace, err)
```
Comment (sergioifg94) [inline test/common/pvc_validate.go:57]:
```suggestion
			t.Errorf("Error getting PVCs for namespace: %v %w", pvcNamespace.Namespace, err)
			continue
```

`t.Errorf` won't stop the test execution, so if there's an error here, the value of `pvcs` won't be set, which is later used in `checkForClaim`. I'd add a `continue` to safely skip this namespace after the error.

File: test/common/pvc_validate.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+			_, err := checkForClaim(claim, pvcs)
+			if err != nil {
+				t.Errorf("Persistant Volume Claim: %v %v", claim, err)
+			}
+
+		}
+	}
+}
+
+func checkForClaim(claim string, pvcs *corev1.PersistentVolumeClaimList) (bool, error) {
```
Comment (sergioifg94) [inline test/common/pvc_validate.go:70]:
```suggestion
func checkForClaim(claim string, pvcs *corev1.PersistentVolumeClaimList) error {
```

As the `bool` result is not used when the function is called, I'd just remove it

File: test/common/pvc_validate.go (hunk 1 above)
Comment (CathalOConnorRH) [reply test/common/pvc_validate.go:57]: updated,  Thanks

File: test/common/pvc_validate.go (hunk 2 above)
Comment (CathalOConnorRH) [reply test/common/pvc_validate.go]: good call. Updated.

**Pull request merged**